Firebase/Messaging/FIRMessagingContextManagerService.m

```diff
@@ -44,7 +44,7 @@
 NSString *const kFIRMessagingContextManagerSoundKey = kFIRMessagingContextManagerNotificationKeyPrefix @"sound";
 NSString *const kFIRMessagingContextManagerContentAvailableKey =
     kFIRMessagingContextManagerNotificationKeyPrefix @"content-available";
-NSString *const kFIRMessagingAPNSPayloadKey = @"aps";
+const NSString *const kFIRMessagingAPNSPayloadKey = @"aps";
```

Comment:
Extra const is unnecessary

Reply:
Sorry, should be static. Didn't copy over correctly from the other branch.